src/app/page.js

```diff
@@ -92,6 +94,15 @@ function Home() {
           </div>
           <div className="bg-[#DCDCE5] dark:bg-[#2F2F3A]"></div>
         </div>
+        <div className="ml-24 w-full h-96 bg-violet-500 p-5">
+          {!items || items?.length === 0 ? (
+            <Welcome />
+          ) : items[0]?.active && items[0].title === "Welcome" ? (
+            <Welcome />
```

Comment:
Could you turn this back to just `<div>` 
And not a custom component. You can give it a color so we know and tell it's working